sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -298,7 +298,7 @@ static void trackEvent(String name, Map<String, String> properties, AnalyticsTra
      * @param transmissionTargetToken A string to identify a transmission target.
      * @return a transmission target.
      */
-    public static AnalyticsTransmissionTarget getTransmissionTarget(String transmissionTargetToken) {
+    public static AnalyticsTransmissionTarget getTransmissionTarget(@NonNull String transmissionTargetToken) {
```

Comment:
No you have to check that, @NonNull does not work for people not having support annotations and can even trigger compile warnings because of missing jar.

Our policy is to use @NonNull only for non public APIs.